docs/advanced-use.rst

```diff
@@ -0,0 +1,157 @@
+.. _advanced-use:
+
+Data pre-processing and troubleshooting
+=======================================
+
+Pre-processing your data
+------------------------
+
+Manipulating the text and/or audio data that you are trying to align can
+sometimes produce longer, more accurate ReadAlongs, that throw less
+errors when aligning. While some of the most successful techniques we
+have tried are outlined here, you may also need to customize your
+pre-processing to suit your specific data.
+
+Audio pre-processing
+~~~~~~~~~~~~~~~~~~~~
+
+Adding silences
+^^^^^^^^^^^^^^^
+
+Adding 1 second segments of silence in between phrases or paragraphs
+sometimes improves the performance of the aligner. We do this using the
+`Pydub <https://github.com/jiaaro/pydub>`__ library which can be
+pip-installed. Keep in mind that Pydub uses milliseconds.
+
+If your data is currently 1 audio file, you will need to split it into
+segments where you want to put the silences.
+
+::
+
+   ten_seconds = 10 * 1000
+   first_10_seconds = soundtrack[:ten_seconds]
+   last_5_seconds = soundtrack[-5000:]
+
+Once you have your segments, create an MP3 file containing only 1 second
+of silence.
+
+::
+
+   from pydub import AudioSegment
+
+   wfile = "appended_1000ms.mp3"
+   silence = AudioSegment.silent(duration=1000)
+   soundtrack = silence
+
+Then you loop the audio files you want to append (segments and silence).
+
+::
+
+   seg = AudioSegment.from_mp3(mp3file)
+   soundtrack = soundtrack + silence + seg
+
+Write the soundtrack file as an MP3. This will then be the audio input
+for your Read-Along.
+
+::
+
+   soundtrack.export(wfile, format="mp3")
+
+Text pre-processing
+~~~~~~~~~~~~~~~~~~~
+
+Numbers
+^^^^^^^
+
+ReadAlong Studio cannot align numbers written as digits (ex. "123").
+Instead, you will need to write them out (ex. "one two three" or "one
```

Comment:
This isn't totally true if G2P has a numbers processing mapping. I think I write about this in the blog so hopefully I remember to link it here as an alternative option.

Reply:
But in most languages it doesn't having such a processing mapping. But yes, if you write about it in the blog, a link here would be great.